dbms/src/Flash/Coprocessor/DAGQueryBlockInterpreter.cpp

```diff
@@ -1039,6 +1111,16 @@ void DAGQueryBlockInterpreter::executeImpl(DAGPipeline & pipeline)
         handleTableScan(table_scan, pipeline);
         dagContext().table_scan_executor_id = query_block.source_name;
     }
+    else if (query_block.source->tp() == tipb::ExecType::TypeWindow)
+    {
+        handleWindow(pipeline, query_block.source->window());
+        recordProfileStreams(pipeline, query_block.source_name);
```

Comment:
Should try to restore concurrency right after window function is executed.

Reply:
> Should try to restore concurrency right after window function is executed.

done